src/sparseml/export/validators.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os.path
+from pathlib import Path
+from typing import List, Union
+
+from sparseml.export.export_data import InputsNames, LabelNames, OutputsNames
+from sparseml.export.helpers import ONNX_MODEL_NAME
+from sparsezoo.inference import InferenceRunner
+from sparsezoo.objects import File, NumpyDirectory
+
+
+__all__ = ["validate_correctness", "validate_structure"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def validate_structure(
```

Comment:
this LGTM - one suggestion for the errors, is to let them accumulate, so rather than failing on the first missing file, we wait to see all missing files and then fail with a message that includes all the missing file names

Reply:
Good idea, will do!